evm/db/backends/level.py

```diff
@@ -0,0 +1,51 @@
+import shutil
+from .snapshot import Snapshot
+from .base import (
+    BaseDB,
+)
+
+
+class LevelDB(BaseDB):
+    db = None
+    db_path = None
+
+    # Creates db as a class variable to avoid level db lock error
+    def __init__(self, db_path='./evm/db/backends/db'):
+        try:
+            import leveldb
+        except ImportError:
+            raise ImportError("LevelDB requires the leveldb \
+                               library which is not available for import.")
+        if not LevelDB.db:
+            self.db_path = db_path
+            LevelDB.db = leveldb.LevelDB(db_path,
+                                         create_if_missing=True, error_if_exists=False)
+
+    def get(self, key):
+        return bytes(self.db.Get(key))
```

Comment:
This type of explanation would be a great comment to include above this line so that future developers who read this source can easily understand *why* since there is no clear context without an existing understanding of the LevelDB API.